include/redisserver.h

```diff
@@ -335,7 +335,7 @@ class RedisServer {
         /*!
         *   \brief Check that the SSDB environment variable
         *          value does not have any errors
-        *   \throw std::runtime_error fi there is an error
+        *   \throw smart_runtime_error fi there is an error
```

Comment:
It must have been introduced in another PR, but "fi" should be "if"

Reply:
fixed